pkg/kubecost/allocation.go

```diff
@@ -107,9 +107,52 @@ type RawAllocationOnlyData struct {
 	RAMBytesUsageMax float64 `json:"ramByteUsageMax"`
 }
 
-// PVUsage contains the byte hour usage
+// PV is a map of Disk Asset Identifiers to the
+// usage of them by an Allocation as recorded in a PVAllocation
+type PV map[PVKey]*PVAllocation
```

Comment:
Sorry to be nitpicky, but this name being singular looks misleading now. "PVs" or "PVAllocations" might be better imo. I like the way that it looks with that name on the `Allocation` type. What do you think?